### PR TITLE
Bugfix for diary and removal of PA book from vendors.

### PR DESCRIPTION
**Pull request by PhoronRestructurantHuman**
## About The Pull Request
You now won't be able to pick a trait that doesn't exist, and you will no longer be able to purchase a PA book for 600 caps.

## Why It's Good For The Game
PA should require training, or at least a bit of effort. Right now you just get your PA, click a trashpile and buy the book. Now you'll have to actually find the book itself AND a powerarmor (But those are everywhere). Bugfix is a bugfix

## Pre-Merge Checklist
- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog

/:cl:
tweak: PA book removals
:cl:

### CODE REVIEW

**Comment by PhoronRestructurantHuman** [comment]
I'm going to fucking cry

File: code/modules/WVM/wvm.dm [hunk 1]
```diff
@@ -679,14 +679,14 @@ GLOBAL_VAR_INIT(vendor_cash, 0)
 		new /datum/data/wasteland_equipment("Random manual",					/obj/item/book/manual/random,									40),
 		new /datum/data/wasteland_equipment("Box of ingredients - American",	/obj/item/storage/box/ingredients/american,						80),
 		new /datum/data/wasteland_equipment("Music box",						/obj/item/holodisk/musicbox,									400),
-		new /datum/data/wasteland_equipment("Advanced Armor and You",			/obj/item/book/granter/trait/pa_wear,							600),
+		new /datum/data/wasteland_equipment("Grognak the Barbarian",			/obj/item/book/granter/trait/bigleagues,						600),
```
Comment (hecksadecimal) [inline code/modules/WVM/wvm.dm:682]: Adding a book to the vendor list is a little out of scope for a removal PR. I'd prefer this be added in a different PR so that they remain atomic and either can be easily reverted individually should the need arise.

**Review by hecksadecimal — CHANGES_REQUESTED** [review]

File: code/modules/WVM/wvm.dm (hunk 1 above)
Comment (PhoronRestructurantHuman) [reply code/modules/WVM/wvm.dm:682]: no

Comment (hecksadecimal) [reply code/modules/WVM/wvm.dm]: I'll remove the lines myself then.

Comment (PhoronRestructurantHuman) [reply code/modules/WVM/wvm.dm]: It's such a minor thing, and forcing people to only be able to obtain Big Leagues by using the diary is silly, it'll practically force you to pick Grognak if you want to do a melee build (which also requires trekking)

**Review by hecksadecimal — APPROVED** [review]

File: code/modules/WVM/wvm.dm (hunk 1 above)
Comment (hecksadecimal) [reply code/modules/WVM/wvm.dm]: I'm not telling you that it's a bad idea, I'm telling you to make two separate PRs since these are two separate things.
